src/bramble/client/StateManager.js

```diff
@@ -106,9 +106,13 @@ define(function() {
                 get: function()  { return getBool(storage, "allowJavaScript"); },
                 set: function(v) { storage.setItem(prefix("allowJavaScript"), v); }
             },
+            editHint: {
+                get: function()  { return getBool(storage, "editHint"); },
+                set: function(v) { storage.setItem(prefix("editHint"), v); },
             autoUpdate: {
```

Comment:
this is still missing a `},` though: the opening `{` for the `editHint` property is not matched at the moment. So if you apply autoformatting, `autoUpdated` is currently *part of* your `editHint` object, which cannot be the intention